make/libraries

```diff
@@ -123,22 +123,23 @@ ifeq (,$(TBB_CC))
   $(error "Need to set TBB_CC to C compiler command for non-standard compiler other than gcc or clang.")
 endif
 
-tbb-make-check:
+$(TBB_BIN)/tbb-make-check:
 	@if [[ $(OS) == Windows_NT ]]; then \
 		if ! [[ $(MAKE) =~ mingw32 ]]; then \
-			echo "Error: Please use mingw32-make on Windows. This is packaged with RTools, for example."; \
+			echo "ERROR: Please use mingw32-make on Windows to build the Intel TBB library."; \
+			echo "This is packaged with RTools, for example."; \
 			exit 1; \
```

Comment:
Could this just throw a warning? There are other make versions than mingw32-make which work with CmdStan

Reply:
but not with the Intel TBB. At least I don't know any other make which can build the TBB. An error is much preferable, since it will stop the whole thing at that point so that the user actually sees the message.

Have you tried other make's which can build the TBB on Windows?

(BTW... if you make a touch on the tbb-make-check file you can circumvent the check)

Reply:
... and BTW... once the TBB is build, you can revert to the standard `make` - even on Windows. The `mingw32-make` is really only mandatory for the Intel TBB build.